azure-kusto-data/azure/kusto/data/client.py

```diff
@@ -199,7 +199,7 @@ def execute_streaming_ingest(
         self,
         database: Optional[str],
         table: str,
-        stream: IO[AnyStr],
+        data: Union[IO[AnyStr], str],
```

Comment:
Not a big fan of overloading this argument.
Why not add another parameter for blob uri? Or a different method?

Even if not so, from looking at the type it seems like it would ingest what's in the str, not a blob that the str points to, you might want a better class.